lib/cassava/client.rb

```diff
@@ -17,8 +17,8 @@ def insert_async(table, data)
 
     # @param [Symbol] table the table name
     # @param [Hash] A hash of column names to data, which will be inserted into the table
-    def insert(table, data)
-      statement = insert_statement(table, data)
+    def insert(table, data, options = {})
```

Comment:
@desmondyeung Unlike `select` and `delete`, `insert` does not actually use the `StatementBuilder` (although it possibly should!), so that actually will not work.